css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,53 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
```

Comment:
Unfortunately it looks like the title in the example you posted is not centered (it's roughly 30px off to the right). This is the reason why I didn't go for that approach in the first place, and I used position: absolute instead.